app/src/main/java/edu/uc/reedws/musiclink/dao/AppDatabase.kt

```diff
@@ -5,6 +5,6 @@ import androidx.room.RoomDatabase
 import edu.uc.reedws.musiclink.dto.PlaylistDTO
 
 @Database(entities = [PlaylistDTO::class], version = 1)
-abstract class AppDatabase: RoomDatabase() {
+abstract class AppDatabase : RoomDatabase() {
```

Comment:
I personally prefer only putting a space after the colon... not sure if this approach is somehow better. @reediculous456 @romigben what do you guys think, since we will be working on this code, and if I write something I will instinctively always put a space after and remove the space before.

Reply:
This is actually the Android Studio auto format that did this

Reply:
Control-Alt-L.